client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -108,22 +108,28 @@ export class GoogleAnalyticsForm extends Component {
 		const wooCommerceActive = wooCommercePlugin ? wooCommercePlugin.active : false;
 
 		const nudgeTitle = siteIsJetpack
-			? translate(
-					'Connect your site to Google Analytics in seconds with Jetpack Premium or Professional'
-			  )
+			? translate( 'Connect your site to Google Analytics' )
 			: translate( 'Connect your site to Google Analytics in seconds with the Premium plan' );
 
-		const nudge = shouldShowOfferResetFlow() ? null : (
+		const plan = siteIsJetpack
+			? OPTIONS_JETPACK_SECURITY
+			: findFirstSimilarPlanKey( site.plan.product_slug, {
+					type: TYPE_PREMIUM,
+					...( siteIsJetpack ? { term: TERM_ANNUALLY } : {} ),
+			  } );
+
+		const nudge = (
 			<UpsellNudge
-				description={ translate(
-					"Add your unique tracking ID to monitor your site's performance in Google Analytics."
-				) }
+				description={
+					siteIsJetpack
```

Comment:
Does this catch Atomic sites too?

Reply:
Yes, `siteIsJetpack()` returns true if site is a Jetpack site (atomic), false if the site is hosted on WordPress.com (simple site). I could be mistaken, but I think **Google Analytics options are different on AT sites** (#36332) is the desired behavior and not a bug.

Reply:
Yep! You'll want options to be different but messaging might need to be adjusted between Jetpack & Atomic. Note e.g. if you're linking to Jetpack support docs or wpcom ones.

Reply:
Ohh, I see, yes. 🙂   Thanks @simison, ok will do 👍